### DISCUSSION TITLE
Can we run figma-export on Linux?

### DISCUSSION THREAD

**Discussion by tonymontgomery** [RedMadRobot/figma-export#158] [Q&A] [answered]
I have successfully ran figma-export on MacOS.
I can not get it to run on Ubuntu Linux. Is this possible? 
Or can it be compiled from source to run on Linux?

Would be nice because my Android development is on Linux.

**Comment by subdan** [comment, marked as answer]
Hi. FigmaExport doesn't support Linux right now. I think it‘s possible to add Linux support. I’ll try to add Linux support.

**Comment by subdan** [comment]
I've tried to add linux support and faced with issue that Swift on Linux doesn't support networking (`URLSession` and friends). So, it will take a lot of time to add Linux support. If more people vote for this feature I'll think about adding Linux support.

**Reply by tonymontgomery** [reply]
@subdan  - Thank you for looking into it. I appreciate it.

**Comment by subdan** [comment]
Hello, @tonymontgomery. I'm adding support for Linux and have a couple of questions. Which version of Ubuntu do you use? Which processor architecture do you have: ARM or AMD64(x86_64)?
